admissions/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
```

Comment:
Missing migration files, need to run `make migrations` and `make migrate`. FIles need to be committed as well.